back/lib/factory.rb

```diff
@@ -8,7 +8,7 @@ def self.instance
 
   def participation_method_for(project)
     participation_context = ::ParticipationContextService.new.get_participation_context(project)
-    return ::ParticipationMethod::None.new unless participation_context
+    return ::ParticipationMethod::Ideation.new(project) unless participation_context
```

Comment:
Does it mean `ParticipationMethod::None` is not used anymore? Should we delete it?